docs/node/guides/storybook-plugin.md

```diff
@@ -0,0 +1,12 @@
+# Storybook
+
+![Storybook logo](/shared/storybook-logo.png)
+
+Storybook is a development environment for UI components. It allows you to browse a component library, view the different states of each component, and interactively develop and test components.
+
+## Links to Further Instructions
+
+Storybook doesn't make much sense in the context of Node, but here are links on further details to using Storybook with your frontend framework of choice:
```

Comment:
I would just omit the storybook overview for the node flavor

Reply:
Hey @isaacplmann - the issue we were having was:
+ the following pages *could* be accessed while in "Node" mode:
  + "docs/shared/nx-plugin.md"
  + "docs/shared/react-plugin.md"
  + "docs/shared/workspace/buildable-and-publishable-libraries.md"
+ These pages had the link: "/{{framework}}/storybook/overview" which translates to "/node/storybook/overview"
+ So if I remove the overview, these pages will have broken links.

It seemed the best solution (optimizing for brevity) 0was to add a storybook overview for node (that just pointed to angular/react) -